__tests__/BleManager.js

```diff
@@ -115,6 +127,22 @@ test('When BleManager readRSSI is called it should call BleModule readRSSI', ()
   expect(Native.BleModule.readRSSIForDevice).toBeCalledWith('id', 'transaction')
 })
 
+test('When BleManager calls async function which throws it should return Unknown Error', async () => {
+  Native.BleModule.readRSSIForDevice.mockImplementationOnce(async () => {
+    throw new Error('Unexpected error2')
+  })
+  await expect(bleManager.readRSSIForDevice('id')).rejects.toThrowError(
+    'Unknown error occurred. This is probably a bug!'
```

Comment:
will the original error be also attached?

Reply:
I'll rewrite this test to check more fields.